src/openzaak/components/zaken/tests/test_notifications_send.py

```diff
@@ -1124,6 +1126,135 @@ def test_send_notif_update_zaak_eigenschap(self, mock_notif):
             },
         )
 
+    @override_settings(ENABLE_CLOUD_EVENTS=True)
+    @patch("openzaak.components.zaken.api.cloud_events.CloudEventConfig.get_solo")
+    def test_send_zaak_gemuteerd_cloud_event(self, mock_get_solo, mock_notif):
+        mock_get_solo.return_value = MagicMock()
+        zaak = ZaakFactory.create()
+        get_operation_url("zaak_read", uuid=zaak.uuid)
+
+        with patch(
+            "openzaak.components.zaken.api.cloud_events.send_cloud_event.delay"
+        ) as mock_send:
+            response = self.client.patch(
+                get_operation_url("zaak_update", uuid=zaak.uuid),
+                data=json.dumps({"toelichting": "Updated toelichting"}),
+                content_type="application/json",
+                HTTP_ACCEPT="application/json",
+                HTTP_ACCEPT_CRS="EPSG:4326",
+                HTTP_CONTENT_CRS="EPSG:4326",
```

Comment:
You can use `from openzaak.components.zaken.tests.utils import ZAAK_WRITE_KWARGS` instead:

```suggestion
                **ZAAK_WRITE_KWARGS
```